prober/http.go

```diff
@@ -327,6 +324,19 @@ func ProbeHTTP(ctx context.Context, target string, module config.Module, registr
 			}
 		}
 
+		if resp != nil && !requestErrored {
+			_, err = io.Copy(ioutil.Discard, resp.Body)
+			if err != nil {
+				level.Info(logger).Log("msg", "Failed to read HTTP response body", "err", err)
+				success = false
+			}
+
+			resp.Body.Close()
+		}
+
+		// At this point body is fully read and we can write end time
+		tt.current.end = time.Now()
```

Comment:
This is only setting the end time for the last request

Reply:
Considering that only the last request matters for `transfer` times, should I prefix this:

```go
durationGaugeVec.WithLabelValues("transfer").Add(trace.end.Sub(trace.responseStart).Seconds())
```

with this:

```go
if trace.end.IsZero() {
	continue
}
```

or do you have other suggestions?

Reply:
That sounds reasonable to me, sound okay to you @discordianfish?

Reply:
I've pushed the change speculatively.

Reply:
Sounds reasonable